packages/catalog-server/src/lib/firestore/firestore-repository.ts

```diff
@@ -441,6 +489,32 @@ export class FirestoreRepository implements Repository {
     return customElementsResults.docs.map((d) => d.data());
   }
 
+  async queryElements({
+    query,
+    limit,
+  }: {
+    query?: string;
+    limit?: number;
```

Comment:
Can limit be non-optional here, since the same `25` default is assigned earlier in the call stack? Or maybe factored out to a constant? (Just thinking if we want to change the default, it would be easy to only do it in one place).

Reply:
It actually seems fine for it to differ. The graphql one is what matters most. This is just a fallback incase someone called the repo directly without a limit. Required is fine too, though in a JS world I'd still usually check before launching a non-limited query :)